 playerop.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git playerop.cpp playerop.cpp
index e025c80..9f8beca 100644
--- playerop.cpp
+++ playerop.cpp
@@ -825,7 +825,7 @@ int32 field::announce_attribute(int16 step, uint8 playerid, int32 count, int32 a
 	}
 	return TRUE;
 }
-static int32 is_declarable(card_data const& cd, const std::vector<uint32>& opcode) {
+int32 is_declarable(card_data const& cd, const std::vector<uint32>& opcode) {
 	std::stack<int32> stack;
 	for(auto& it : opcode) {
 		switch(it) {
